docs/advanced/advanced.md

```diff
@@ -192,6 +192,49 @@ The contents of `extra_navbar` will be inserted into your page's HTML after
 all other html content.
 
 
+## Working on Windows
+
+As of May 27, 2020, there are three open issues that require Windows-specific workarounds:
+
+1. Character encoding
+
+    Jupyer-book current reads and writes files on windows in the native windows encoding, which causes encoding errors for some characters in UTF-8 encoded notebooks.
+
+    **Work-around:**  Beginning with [Python 3.7](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUTF8) cmd.exe or powershell enviroments that set PYTHONUTF8=1  override the native locale encoding and use UTF8 for all input/output.  To make it easier to use this option, we have created a conda package [runjb](https://anaconda.org/eoas_ubc/runjb) which [does this automatically for powershell](https://github.com/eoas-ubc/eoas_tlef/blob/master/converted_docs/wintools/binwin/runjb.ps1)
```

Comment:
Could you try capping line lengths around 80 characters and splitting long lines across multiple ones?